dev/devicelab/bin/tasks/hot_mode_dev_cycle__benchmark.dart

```diff
@@ -10,18 +10,22 @@ import 'package:flutter_devicelab/framework/adb.dart';
 import 'package:flutter_devicelab/framework/framework.dart';
 import 'package:flutter_devicelab/framework/utils.dart';
 
-void main() {
+void main({bool isPreviewDart2: false}) {
```

Comment:
This code is becoming more and more reusable. Could you move it under `lib/tasks`?

I'm worried that one day someone might edit this file, add `deviceOperatingSystem = DeviceOperatingSystem.ios;` (which is a common pattern in many other benchmarks), and become a sad developer.

Reply:
Done